node/src/main_chain_follower.rs

```diff
@@ -50,8 +50,13 @@ fn use_mock_follower() -> bool {
 
 pub fn create_mock_data_sources(
 ) -> std::result::Result<DataSources, Box<dyn Error + Send + Sync + 'static>> {
+	let mv_epoch_duration_millis: u32 = env::var("MC__EPOCH_DURATION_MILLIS")
+		.ok()
+		.and_then(|v| v.parse::<u32>().ok())
+		.unwrap();
+	let block_data_source_mock = BlockDataSourceMock::new(mc_epoch_duration_millis);
```

Comment:
Actually it would be nice to move reading the env::var to mock itself as we strive to have more code in library crates (so all crates except runtime and node).

This should make it easier to integrate PC in other projects.

Reply:
I guess the best way to do this would be to add Figment to the `BlockDataSourceMock` struct? Or to create a new `BlockDataSourceMockConfig` struct that can build a `BlockDataSourceMock`?

From an application perspective, I don't want a library forcing me to set an env-var, but having the option by having some config struct that can optionally be instantiated by env-vars is the best of both worlds.